### PR TITLE
User can create poll #1

### PR DESCRIPTION
**Pull request by rickbacci**
@stevekinney

What basic goals are you looking for tomorrow. I'm going to work for a few hours and see how far I can get.

### CODE REVIEW

File: public/client.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+var submitPoll = document.getElementById('submit-poll');
+
+submitPoll.addEventListener('click', function(event) {
+  event.preventDefault();
+
+  var question = document.getElementById('question').value;
+  var choiceA  = document.getElementById('choice-A').value;
+  var choiceB  = document.getElementById('choice-B').value;
+  var choiceC  = document.getElementById('choice-C').value;
+  var choiceD  = document.getElementById('choice-D').value;
```
Comment (stevekinney) [inline public/client.js]: You'll probably want to take a variable number of choices.

File: public/client.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  var poll = {
+    question: question,
+    choiceA: choiceA,
+    choiceB: choiceB,
+    choiceC: choiceC,
+    choiceD: choiceD,
+  }
+  console.log(poll);
+  socket.send('pollCreated', poll)
```
Comment (stevekinney) [inline public/client.js]: It might make sense to just use a regular old `POST` request.

File: public/client.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+});
+
+
+var adminUrl = document.getElementById('admin-url');
+var voterUrl = document.getElementById('voter-url');
+var baseUrl  = 'http://localhost:3000'
```
Comment (stevekinney) [inline public/client.js]: You probably don't want this hard-coded in because it won't work in production.

File: public/client.js (hunk 2 above)
Comment (rickbacci) [inline public/client.js]: I thought that using ajax or sockets was the same either way. Does one have a benefit over the other?

File: public/client.js (hunk 1 above)
Comment (rickbacci) [inline public/client.js]: I can do that. I read the spec as 3-4 choices, but variable will be better.

**Pull request merged**